cockroachcloud/cloud-api.md

```diff
@@ -390,6 +390,46 @@ Where:
 
 If the `DELETE` request was successful the client will not receive a response payload.
 
+<a id="cloud-audit-logs"></a>
+
+## Export {{ site.data.products.db }} audit logs
+
+{% include feature-phases/preview-opt-in.md %}
+
+To export audit logs for activities and events related to your {{ site.data.products.db }} organization, send a `GET` request to the `/v1/auditlogevents` endpoint. The service account associated with the secret key must have `ADMIN` [permission](console-access-management.html#service-accounts).
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url 'https://cockroachlabs.cloud/api/v1/auditlogevents?StartingFrom={timestamp},SortOrder={sort_order},limit={limit}' \
+  --header 'Authorization: Bearer {secret_key}'
+~~~
+
+Where:
+
+  - `{timestamp}` indicates the first log entry to fetch. If unspecified, `{timestamp}` is set to the beginning of time if `{sort_order}` is `ascending`, or the current time if `{sort_order}` is `descending`.
```

Comment:
Should we also refer to the expected format of "timestamp" here, explaining the different parts of it - year, month, day, hour, minute, seconds, microseconds?

Reply:
This is the same time format as every other timestamp in the CC API, formatted as RFC3339 format https://www.ietf.org/rfc/rfc3339.txt. There's a detailed explanation here https://github.com/protocolbuffers/protobuf/blob/7d1362c7ba96ef13de713869513ca32d79a02931/src/google/protobuf/timestamp.proto#L108.

Reply:
Got it, thanks. I think we should explain this because I think this is the first time an API would expose date/time as an input parameter.